### PR TITLE
Updated docs to match Taqueria website

### PR DESCRIPTION
**Pull request by alexzbusko**

### CODE REVIEW

**Review by hu3man — APPROVED** [review]
Awesome!

File: README.md [hunk 1]
```diff
@@ -1,21 +1,129 @@
...
+- Docker v20.10.12 or later
+- Node.js v16.17.1 or later
+- Temple Wallet (can be found [here](https://templewallet.com/))
+- A funded testnet account (instructions [here](https://taqueria.io/docs/config/networks/#faucets))
+
+## Using the Project
+
+### Adding Artwork Assets
+
+To add artwork and minting it's token:
```
Comment (claudebarde) [inline README.md]: Shouldn't it be "its token"?

File: README.md [hunk 2]
```diff
@@ -1,21 +1,129 @@
...
+
+1. Add the artwork asset file (image, video, etc) to the assets directory
+    - /taqueria/assets
+    - Make sure to name the asset file with the next token id: 
+    - i.e. `0042.jpg`
+1. Add a thumbnail image for the nft to the same assets directory
+    - use the filename with the same token id with `.thumb`
+    - the thumbnail image does not need to be the same file type as the main asset
+    - i.e. `0042.thumb.png`
+1. Publish the artwork asset to ipfs and mint the nft to the contract by running the provisioner
```
Comment (claudebarde) [inline README.md]: There is also a general inconsistency throughout the text between "nft" and "NFT", I think the latter is better as it is an acronym

File: README.md [hunk 3]
```diff
@@ -27,12 +135,47 @@
...
+        - The compiled contract (*.tz file)
+    - `taqueria/types`
+        - The contract typescript typing
+    - `taqueria/assets`
+        - The NFT assets to upload
+
+### React Dapp
+
+The react Dapp allows browsing the NFTs that have been minted to the contract. 
+This is a minimal react project and needs to be extended to support marketplace features like selling or browsing owned NFTs.
```
Comment (claudebarde) [inline README.md]: It should be "React" with a capital R

**Review by claudebarde — APPROVED** [review]

**Pull request merged**